example_config/gcp-auth-hook.py

```
@@ -0,0 +1,162 @@
+
+
+def mint_token():
+    global _cached_token, _cached_expiry
+
+    now = time.time()
+    if _cached_token and now < (_cached_expiry - REFRESH_SKEW_SECONDS):
+        return _cached_token
+
+    token_cmd = os.environ.get(TOKEN_CMD_ENV, DEFAULT_TOKEN_CMD)
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)